.github/workflows/ci-e2e-vrt.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     playwright:
         name: Visual Regression Tests
         runs-on: ubuntu-20.04
+        container:
+            image: mcr.microsoft.com/playwright:v1.28.1-focal
```

Comment:
nit: the only thought I'd have, is to add a comment to the package.json at the playwright line to point people to updating this line...

Reply:
unfortunately package.json doesn't support json comments.. i frequently find myself wanting to add comments there for exactly reasons like this. best alternative i've found so far is to keep a rolling issue (i.e. have comments regarding the individual packages there and copy them over to the pr for the next upgrade) for upgrades.